src/DynamoCore/Core/NodeModelAssemblyLoader.cs

```diff
@@ -199,6 +199,44 @@ internal static bool ContainsNodeModelSubType(Assembly assem)
             return assem.GetTypes().Any(IsNodeSubType);
         }
 
+        internal static bool ContainsNodeModelSubTypeReflectionLoaded(Assembly assem, Type nodeModelType = null)
+        {
+            if(!(nodeModelType is null))
```

Comment:
This might be easier to read instead of negating a is null assertion
```suggestion
            if(nodeModelType is null)
                return assem.GetTypes().Any(x => IsNodeSubType(x));
            return assem.GetTypes().Any(x => IsNodeSubTypeReflectionLoaded(x, nodeModelType));
            
```